samtranslator/intrinsics/resolver.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, parameters, supported_intrinsics=None):
             )
 
         if not isinstance(supported_intrinsics, dict) or not all(
-            [isinstance(value, Action) for value in supported_intrinsics.values()]
+            (isinstance(value, Action) for value in supported_intrinsics.values())
```

Comment:
Shouldn't this be:

```suggestion
            isinstance(value, Action) for value in supported_intrinsics.values()
```

Reply:
No, the inner part needs to be an iterable.